gwsumm/plot/core.py

```diff
@@ -324,6 +324,8 @@ def pid(self):
             filts = "".join(map(str,
                 [getattr(c, 'filter', getattr(c, 'frequency_response', ''))
                  for c in self.channels]))
+            if self.filterstr:
+                filts = "".join([filts,self.filterstr])
```

Comment:
sort of, you can use `super()` to call the parent `pid()` method and then append in the same manner that is done [here](https://github.com/tjma12/gwsumm/blob/34fa68da90718adf544448d967a6bc6745869e7a/gwsumm/plot/builtin.py#L260)